ConservationInternational/resilienceatlas
# RA2-24: Fixes of journeys at Admin

Bunch of fixes related to journeys based on result of QA:
 - fix: validate presence of journey steps at Journey model (at least one step is required per journey)
 - refactoring: simplify trix editor toolbar for journeys at Admin (no need for attachments, code, etc. options)
 - fix: invalidate cache when translation of model changed (touch original record)
 - refactoring: adding `background_color` to `chapter` journey step

FYI @Bluesmile82 

## Testing instructions

It should be checked that Journeys can be still created via Admin and that:
 - cannot be created without at least one step
 - wysiwyg editor does not allow to add attachments (admin see simplified version of it)
 - cache is invalidated when for example title of journey changes (response returned by API is updated)
 - `background_color` attribute is available for chapter journey step

## Tracking

https://vizzuality.atlassian.net/browse/RA2-24
